scripts/package.unmask/ghc-9.8

```diff
@@ -20,6 +20,7 @@
 <dev-haskell/haddock-library-1.12
 <dev-haskell/primitive-unlifted-2.2
 <dev-haskell/req-3.14
+<dev-haskell/sbv-12
```

Comment:
Could you please also copy this line to `scripts/package.unmask/ghc-9.10` and `ghc-9.12`?